Picker/Picker.js

```diff
@@ -63,6 +65,53 @@ const PickerBase = kind({
 		 */
 		children: PropTypes.array.isRequired,
 
+		/**
+		 * Prevents read out of both the slider and the increment and decrement
+		 * buttons.
+		 *
+		 * @type {Boolean}
+		 * @memberof agate/Picker.PickerBase.prototype
+		 * @public
+		 */
+		'aria-hidden': PropTypes.bool,
```

Comment:
Please check if we really need to define as a prop. This is the same as dom attribute and react can pass to the dom